pkg/core/resources/apis/mesh/zz_generated.resources.go

```diff
@@ -1024,7 +1024,7 @@ var MeshGatewayResourceTypeDescriptor = model.ResourceTypeDescriptor{
 	KumactlArg:          "meshgateway",
 	KumactlListArg:      "meshgateways",
 	AllowToInspect:      false,
-	IsPolicy:            true,
+	IsPolicy:            false,
```

Comment:
Might be worth mentioning in UPGRADE.md since it's an API change?

Reply:
since it's not used in GUI nor in kumactl, I don't think it's worth it